src/styles/util.scss

```diff
@@ -89,3 +89,11 @@
 .u-color-text--muted {
   color: var.$color-text--light;
 }
+
+// TODO: The sidebar should really apply vertical spacing to its immediate children,
```

Comment:
In this context it isn't clear to me which element is meant by "the sidebar" and what qualifies as a "container" that should use this element. My understanding is that the purpose of this is to apply a gap between logical sections of the sidebar, where the panel (if any) at the top counts as a "section" and the tabs + thread list count as another "section". Is that right?

I also noticed while looking at this that `FilterStatus` has the same margin visually and looks like a `Panel` but it doesn't use `Panel`. Should it?

Reply:
Need to take another look at `FilterStatus`, definitely.

Reply:
I've completely re-written this comment.